main.js

```diff
@@ -0,0 +1,88 @@
+console.log("It can't rain all the time");
+
+// grabbng DOM nodes
+var btn = document.querySelector('button');
+var pCurrentWork = document.querySelector('p');
+var number = 10; //gonna be 25*60
+var number2 = 5;
+var task = document.querySelector('input');
+var timer = document.querySelector('#timer');
+var taskContainer = document.querySelector('.task-container');
+var configContainer = document.querySelector('.config-container');
+var configBtn = document.querySelector('.config');
+var timerId = null;
+var timerId2 = null;
+
+
+// create Start Timers
+ var handleStart = function() {
+    timerId = setInterval(startTimer, 1000);
+    pCurrentWork.innerHTML = pCurrentWork.textContent + task.value;
+    btn.removeEventListener('click', handleStart);
+  };
+
+var handleStart5min = function() {
+  timerId2 = setInterval(startFiveMinTimer, 1000);
+  };
+
+
+// function to start a timer
+var startTimer = function() {
+  number = number - 1;
+  if (number === 0) {
+    clearInterval(timerId) // stop the timer
+    pCurrentWork.innerHTML = "Completed tasks: " + task.value;
+    var takeBreak = document.createElement('p');
+    takeBreak.setAttribute('id','break');
+    takeBreak.innerHTML = 'Take a break!';
+    taskContainer.appendChild(takeBreak);
+    handleStart5min();
+    timerId = null;
+    number = 10;
+  };
+    timer.textContent = number + ' in seconds';
+};
+
+// function to start a five min timer
+var startFiveMinTimer = function() {
+  number2 = number2 - 1;
+  if (number2 === 0) {
+    clearInterval(timerId2);
+  pCurrentWork.innerHTML = '';
```

Comment:
Make sure that all of this is indented properly on the same line, otherwise it's easy to confuse what work is happening inside the conditional.